### PR TITLE
Made additions in "Community Values" section of Community tab

### PR DESCRIPTION
**Pull request by aj11anuj**
Made some additions in "Community Values" section of [Community tab](https://kubernetes.io/community/)

### CODE REVIEW

**Review by nate-double-u — COMMENTED** [review]
Hi @aj11anuj, does this PR relate to an issue or conversation I've missed (and I apologize if I have)? If so, could you link it here?

I think that updating the community values should involve community discussion and likely a triage/accepted issue.

**Comment by aj11anuj** [comment]
> Hi @aj11anuj, does this PR relate to an issue or conversation I've missed (and I apologize if I have)? If so, could you link it here?
> 
> I think that updating the community values should involve community discussion and likely a triage/accepted issue.

Hello Nate 👋,
You hasn't missed any conversation related to this PR as it's not discussed anywhere, and this PR is not linked with any issue.

I haven't added these points on my own, i have referenced these points from K8s Contributor site "Community Values" page (This page is linked to 'Read More' button, that is located just below where i have made additions). So, basically i just referenced these points from that page and added it here in brief as earlier it was looking a little empty empty 😅

File: content/en/community/_index.html [hunk 1]
```diff
@@ -61,7 +61,8 @@
 <div class="community-section" id="values">
   <h2>Community Values</h2>
   <p>The Kubernetes Community values are the keystone to the ongoing success of the project.<br class="optional"/>
-  These principles guide every aspect of the Kubernetes project.</p>
+   These principles guide every aspect of the Kubernetes project. These values have evolved over time, pushing our project and peers toward constant improvement of it.
+   "Automation over Process", "Community over Company", "Distribution over Centralization" are some of our main Community Values. Read more about our Community Values from here.</p>
```
Comment (Gauravpadam) [inline content/en/community/_index.html:65]:
```suggestion
   Our key Community Values include "Prioritizing Automation over Process," "Preferring Community over Company," and "Emphasizing Distribution over Centralization." We encourage our readers to know more about these values
```

**Review by Gauravpadam — COMMENTED** [review]
Just a few edits, Rest lgtm

**Comment by Gauravpadam** [comment]
Of course, If @nate-double-u suggests a triage or some discussion prior to the updates, We should follow it :)

**Comment by aj11anuj** [comment]
> Of course, If @nate-double-u suggests a triage or some discussion prior to the updates, We should follow it :)

I think a prior discussion was required when this PR was about to make some "new additions" in Community Values, that wasn't written-ed yet anywhere about K8s community. But here there aren't any new additions, just some brief community values are referenced from one K8s page to another K8s page. Do it needs a prior discussion? 🤔

**Comment by nate-double-u** [comment]
> I haven't added these points on my own, i have referenced these points from K8s Contributor site "Community Values" page (This page is linked to 'Read More' button, that is located just below where i have made additions). So, basically i just referenced these points from that page and added it here in brief as earlier it was looking a little empty empty 😅

So the way I read it,  https://www.kubernetes.dev/community/values/ is the source of truth in this, and https://kubernetes.io/community/values/ is a mirror of that page with instructions on how to update copy if it is out of date.

I don't think we should or need to update the copy here.

**Comment by nate-double-u** [comment]
Thanks for the effort, @aj11anuj, but I'm going to close this. If you do think this text should be updated and expanded on the community page, let's open an issue to discuss how best to do that.

/close

**Pull request closed without merging**